recipe/meta.yaml

```diff
@@ -4,6 +4,7 @@
 {% set bootstrapped_wheels = [
   {"name": "pip", "version": "23.3.2", "sha256": "5052d7889c1f9d05224cd41741acb7c5d6fa735ab34e339624a614eaaa7e7d76"},
   {"name": "setuptools", "version": "69.0.3", "sha256": "385eb4edd9c9d5c17540511303e39a147ce2fc04bc55289c322b9e5904fe2c05"},
+  {"name": "wheel", "version": "0.42.0", "sha256": "177f9c9b0d45c47873b619f5b650346d632cdc35fb5e4d25058e09c9e581433d"},
```

Comment:
Seeing the following error [on CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=869956&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3 ). Also including [the attached log]( https://github.com/conda-forge/certifi-feedstock/files/14150456/certifi-feedstock_linux-64_20240203.7.txt ) for posterity

```
 error: invalid command 'bdist_wheel'
```

Adding `wheel` to the bootstrapped dependencies to fix this issue